en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+                <td>Enable account lock during self registration. The account will be unclocked 
+                upon confirmation.</td>
+            </tr>
+            <tr class="odd">
+                <td>Enable Notification Internally Management</td>
+                <td>
+                    <p>
+                        Select to configure Identity server to send confirmation emails to the user.
+                        If the client application handles notification sending already, unselect it. 
+                    </p>
+                </td>
+            </tr>
+            <tr class="even">
+                <td>Enable reCaptcha</td>
+                <td>Select to enable reCaptcha for self-registration. 
+                See <a href="../../learn/configuring_google_recaptcha_for_security_question_based_password_recovery">
+                Configuring Google reCaptcha for Security-Question Based Password Recovery</a> for more information.</td>
+            </tr>
+            <tr class="odd">
+                <td>User self registration verification link expiry time.</td>
+                <td>
+                    <div class="content-wrapper">
+                    <p>Number of minutes that the confirmation link would be valid. The confirmation link 
+                    will expire after the specified time has elapsed.</p>
+                        <div class="admonition note">
+                        <p class="admonition-title">Note</p>
+                        <p>Alternatively, you can configure the expiry time from the <code>deployment.toml</code>  file.</p>
+                        <p>
+                            <code>
+                            [identity_mgt.user_self_registration]<br>
+                            allow_self_registration= true<br> 
+                            expiry_time="1440"
+                            </code>
+                        </p>
+                        </div> 
+                    </div>
+                </td>
+            </tr>
+            <tr class="even">
+                <td>User self registration SMS OTP expiry time</td>
+                <td>
+                    <ul>
+                        <li>User self registration verification link expiry time.</li>
+                        <li>The <b>default and minimum</b> value is <code>1</code> minute.
+                    <ul>
+                </td>
+            </tr>
+        </tbody>
+        </table>
+        
+## Configuring self-registration consent purposes
+    
+For more details see, 
+[Configuring self-registration consent purposes](../.
+./learn/self-registration-and-account-confirmation/#configuring-self-registration-consent-purposes).
+
+## Try out self-registration  
+
+WSO2 Identity Server provides the functionality to confirm the user account internally or externally.
+
+1. **Internal Notification Management** : Manage notification sending an account confirmation 
+by the Identity Server.
+    
```

Comment:
Internal Notification Management : Notification sending for account confirmation is managed by WSO2 Identity Server

External Notification Management: Notification sending and confirming the registration flow to WSO2 Identity Server is managed by an external notification management mechanism.

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549